Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.py

```diff
@@ -37,7 +72,7 @@ def air_acquire(self, hostname: str, profile: str, case_id: str, organization_id
             "taskConfig": {
                 "choice": "use-policy"
             },
-            "acquisitionProfileId": profile,
+            "acquisitionProfileId": self.get_profile_id(profile, organization_id),
```

Comment:
I have a suggestion here, but I wanna make sure what happens if you send the request to the /api/public/acquisitions/acquire without any assigned value ("") in the acquisitionProfileId.
If you come across an API error, instead throwing an error in the get_profile_id function , you can just choose to return a empty string value from it and then you will get an error from the /api/public/acquisitions/acquire endpoint.

If not, first check the output from the get_profile_id function and if the output is empty return a human readable message without context data and with a message 'The acquisition profile "{profile}" cannot be found.' else, create the payload and send the request.